as-ps/systemd/sirius-ioc-as-ps-dclink-ia02.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Power Supply IOC - DCLinks - Instrumentation Area 02
+After=multi-user.target
+
+[Service]
+Group=sirius
+User=sirius
+Type=simple
+WorkingDirectory=/home/sirius/iocs
+StandardOutput=syslog
+StandardError=syslog
+ExecStartPre=/bin/bash --login -c 'env > /tmp/bashrc-sirius-ioc-as-ps-dclink-ia01'
```

Comment:
I think there is a typo here: `dclink-ia01` --> `dclink-ia02`

Reply:
done.